src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -621,9 +769,17 @@ public void setPort(int port) {
     public int getPort() {
         return port;
     }
+    
+    public void setHostAddresses(String hostAddresses) {
```

Comment:
Having one entry per host in the XML configuration looks for better - I agree. However, it makes things more difficult in some cases like Docker container environment.
It would be nice if we could set a single `LOG_ADDRESSES` environment variable with the addresses of log collector hosts in order and simply refer to it in the Logback configuration. Having separate entries per host would require the declare one environment variable for each as well. Do you see the point?

What do you think if we do support both schemes. User would shave three options:
1/ `<remoteHost>`, `<remotePort>`: legacy parameters
2/ collection of `<hostAddress>`: your proposed approach
3/ `<hostAddresses>`: comma separated list

These options would be exclusive.